core/openapi.yaml

```diff
@@ -262,24 +266,15 @@ components:
           format: double
           minimum: 0
           example: 1.05
-        rolling_resistance:
-          oneOf:
-            - $ref: "#/components/schemas/RollingResistanceDavis"
-          discriminator:
-            propertyName: type
-            mapping:
-              davis: "#/components/schemas/RollingResistanceDavis"
-        capabilities:
-          type: array
-          items:
-            type: string
-            enum:
-              - TVM300
-              - TVM430
-              - ETCS1
-              - ETCS2
-              - ETCS3
-              - KVB
+        a:
+          type: number
+          format: double
+        b:
+          type: number
+          format: double
+        c:
+          type: number
+          format: double
```

Comment:
I'm very confused, these fields should match what's in [the railjson schema](https://github.com/DGEXSolutions/osrd/blob/dev/api/osrd_infra/schemas/rolling_stock.py)

Reply:
ok i will check it